addons/event/migrations/9.0.0.1/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from . import test_event
```

Comment:
As travis notes, you do not need to import migration tests.

Reply:
No, that's not the problem I think. I'll tell you later.

Reply:
It is __a__ problem though.
```
$ flake8 addons/*/migrations/*/tests/ --max-line-length=120
addons/event/migrations/9.0.0.1/tests/__init__.py:4:1: F401 '.test_event' imported but unused
The command "flake8 addons/*/migrations/*/tests/ --max-line-length=120" exited with 1.
```